### PR TITLE
Fix #195: Adjust service calls to the String application ID

### LINKED ISSUE #195: Adjust service calls to the String application ID

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/DelegatingActivationCodeHandler.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ public interface DelegatingActivationCodeHandler {
      * @return List of new activation flags for the destination activation.
      * @throws ActivationCodeException Thrown in case activation flag processing fails.
      */
-    List<String> addActivationFlags(String sourceActivationId, List<String> sourceActivationFlags, String userId, String applicationId, Long sourceAppId, List<String> sourceApplicationRoles, Long destinationAppId, String destinationActivationId, String activationCode, String activationCodeSignature) throws ActivationCodeException;
+    List<String> addActivationFlags(String sourceActivationId, List<String> sourceActivationFlags, String userId, String applicationId, String sourceAppId, List<String> sourceApplicationRoles, String destinationAppId, String destinationActivationId, String activationCode, String activationCodeSignature) throws ActivationCodeException;
```
Comment (banterCZ) [inline enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/DelegatingActivationCodeHandler.java:61]: Ok for this PR, but in general, too many arguments of the same type. Very error prone for developers. Could be nice to refactor to a parameter object.

File: mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/entity/AllowedSignatureType.java [hunk 2]
```diff
@@ -37,15 +37,31 @@ public class AllowedSignatureType {
      * Signature types.
      */
     public enum Type {
+
+        /**
+         * Symmetric multi-factor signature with 1FA (possession)
```
Comment (banterCZ) [inline mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/entity/AllowedSignatureType.java:42]: 👍

File: pom.xml [hunk 3]
```diff
@@ -32,7 +32,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.6.6</version>
+        <version>2.6.8</version>
```
Comment (banterCZ) [inline pom.xml:35]: I prefer to keep dependency upgrades in a separate PR.

File: pom.xml [hunk 4]
```diff
@@ -87,19 +87,19 @@
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <maven-deploy-plugin.version>3.0.0-M2</maven-deploy-plugin.version>
         <maven-javadoc-plugin.version>3.3.2</maven-javadoc-plugin.version>
-        <maven-swagger-codegen-plugin.version>3.0.29</maven-swagger-codegen-plugin.version>
+        <maven-swagger-codegen-plugin.version>3.0.34</maven-swagger-codegen-plugin.version>
         <maven-swagger-codegen-handlebars.version>4.3.0</maven-swagger-codegen-handlebars.version>
 
         <db-h2.version>1.4.200</db-h2.version>
```
Comment (banterCZ) [inline pom.xml:93]: h2 is also quite behind and contains CVEs, but fixes are in major version 2.1 :(

**Review by banterCZ — APPROVED** [review]
It could be nice to have dependency upgrade in a separate PR.

**Review by saalistaja — APPROVED** [review]
Looks really good.
